css/login_page.css

```diff
@@ -0,0 +1,42 @@
+.doc-container-login{
+    width: 100%;
+    height: 70vh;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.doc-container-login .form-input label{
+    font-size: 1rem;
+}
+
+.doc-container-login .input-text{
+    font-size: 1rem;
+    margin-bottom: 1.2rem;
+}
+
+.login-form{
+    background-color: var(--white);
+    padding: 3rem;
+    width: 400px;
+}
+
+.form-input h2, h4{
+    text-align: center;
+margin-bottom: 1.2rem;
+}
```

Comment:
Just a minor suggestion : the code formatting could be improved here

Reply:
Yes, will do that.